keyboards/sofle/keymaps/noqmk/keymap.c

```diff
@@ -0,0 +1,686 @@
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+char wpm_str[10];
```

Comment:
```suggestion
```

Reply:
Why exactly should these lines be removed?

Reply:
`wpm_str` is not used any where in keymap.c and including stdio.h isn't necessary.

Reply:
Oh alright, I'll remove it then